intel_extension_for_transformers/neural_chat/docs/notebooks/amp_optimization_on_spr.ipynb

```diff
@@ -18,34 +18,64 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Install intel extension for transformers:"
+    "Install requirements:"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
+    "vscode": {
+     "languageId": "shellscript"
+    }
+   },
    "outputs": [],
    "source": [
-    "!pip install intel-extension-for-transformers"
+    "!git clone https://github.com/intel/intel-extension-for-transformers.git"
```

Comment:
Why using source code installation instead of binary pip installation?

Reply:
Yes, we should use the binary pip installation. Updated.